floodviz/linked_data_utils.py

```diff
@@ -0,0 +1,154 @@
+from datetime import date
+
+
+class LinkedData:
+    """
+    Generates JSON-LD based on gages and the flood event.
+    """
+    def __init__(self):
+        self.ld = self._blank_thing("WebSite")
+        self.ld.update({
+            "name": "Active flood visualization",
```

Comment:
I think we should be getting the name and about through config variables. This might not be ideal in the end workflow but let's start with that.